benchmarks/dashboard-stack/lambda/benchmarkManager.py

```diff
@@ -0,0 +1,68 @@
+import boto3
+
+CODE_BUILD_NAME = "S3BenchmarksDeploy"
+
+
+def benchmarkManager(event, context):
+    '''
+    Lambda handler.
+    Action in event determing how manager runs benchmark stack.
+
+    delete: Delete a stack.
+        - stack_name (string): the name of stack to delete.
+    test: Deploy the stack via code build.
+    '''
+    cf_client = boto3.client('cloudformation')
+    print("## LOG started")
+    print(event)
+    print("## event ends")
+    if 'action' not in event:
+        print("\'action\' is required in the event for BenchmarkManager")
+        return {
+            'statusCode': 400,
+            'headers': {
+                'Content-Type': 'text/plain'
+            },
+            'body': 'event {} is invalid, \
+                \'action\' is required'.format(event)
+        }
+    if event['action'] == 'delete':
+        if 'stack_name' in event:
+            stack_name = event['stack_name']
+        else:
+            print("\'stack_name\' is required for delete action")
+            return {
+                'statusCode': 400,
+                'headers': {
+                    'Content-Type': 'text/plain'
+                },
+                'body': 'event {} is invalid, \
+                    \'stack_name\' is required for delete action'.format(event)
+            }
+        print("Deleting stack, name is {}".format(stack_name))
```

Comment:
Feel like this is dangerous, it can delete the CFN stack in the account. But not sure how can we avoid it....